titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/validator/JobImageValidatorConfiguration.java

```diff
@@ -21,6 +21,9 @@
 
 @Configuration(prefix = "titus.validate.job.image")
 public interface JobImageValidatorConfiguration {
+    @DefaultValue("SOFT")
```

Comment:
This interface looks similar for all validators (has `getErrorType` and `isEnabled`). These operational configuration could be captured via common interface.